File: Euler1.py
# Project Euler
# Problem 1 - Multiples of 3 and 5
# https://projecteuler.net/problem=1
# Engineering 2 9.3.3
# Created by Andrew Voss

# create list for multiples of 3 & 5
i = 0
num = []


# add multiples of 3 to list
while i < 1000:
    num.append(i)
    i += 3

# add multiples of 5 to list
d = 0
while d < 1000:
    num.append(d)
    d += 5

# remove duplicate numbers in list
num1 = list(set(num))

# find sum of multiples of 3 & 5
a = sum(num1)

# print the sum
print "Sum of multiples of 3 & 5 below 1000:", a

File: Euler1.c
/*
 * Euler 1
 * https://projecteuler.net/problem=1
 * Created by Andrew Voss
 */

#include <stdio.h>

#define MAX 1000 // Max value, number to find multiples of 3 and 5

int main()
{
  // Define variables
  int count3, count5, total3, total5, doubles;
  total3 = doubles = 0;

  // Add duplicate multiples to variable "doubles" and find sum of multiples of 3
  for (count3 = 0; count3 < MAX; count3 = count3 + 3){
    total3 = count3 + total3;
    if (count3 % 5 == 0){
      doubles = doubles + count3;
    }
  }

  // find sum of multiples of 5
  for (count5 = 0; count5 < MAX; count5 = count5 + 5)
      total5 = count5 + total5;

  // prints the final total of the multiples of 3 & 5 w/out duplicate multiples
  printf("%d\n", total3 + total5 - doubles);
}
